src/main/java/seedu/address/model/contact/TagsContainKeywordsPredicate.java

```diff
@@ -15,13 +15,11 @@ public TagsContainKeywordsPredicate(List<String> tagKeywords) {
 
     @Override
     public boolean test(Contact contact) {
-        return tagKeywords.stream()
-                .allMatch(keyword ->
-                        contact.getTags().stream()
-                                .anyMatch(tag ->
-                                        StringUtil.containsWordIgnoreCase(tag.tagName, keyword)
-                                )
-                );
+        if (tagKeywords.isEmpty()) {
+            return false;
+        } else {
+            return tagKeywords.stream().allMatch(keyword -> contact.getTags().stream().anyMatch(tag -> StringUtil.containsWordIgnoreCase(tag.tagName, keyword)));
```

Comment:
This could be broken into multiple lines.
```suggestion
            return tagKeywords.stream().allMatch(keyword -> 
                contact.getTags().stream().anyMatch(tag -> 
                    StringUtil.containsWordIgnoreCase(tag.tagName, keyword)));
```